### PR TITLE
New version: Genymobile.scrcpy version 3.0.1

### PR DESCRIPTION
**Pull request by ponces**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/196933)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
> finished with exit code: -2147024893

(Automated response - build 930.)

File: manifests/g/Genymobile/scrcpy/3.0.1/Genymobile.scrcpy.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  InstallerSha256: 2D2485CEAD6BB9D80EC337A660A571FC4B3C2E15034AD73C6A2867442206A5F4
+- Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: scrcpy-win32-v3.0.1\scrcpy.exe
+    PortableCommandAlias: scrcpy
+  InstallerUrl: https://github.com/Genymobile/scrcpy/releases/download/v3.0.1/scrcpy-win32-v3.0.1.zip
+  InstallerSha256: 43296F8BF34DD408C65463D45CA367FEBE68CEC3AE34B78917A8F3ECBF321829
+ManifestType: installer
+ManifestVersion: 1.9.0
+ReleaseDate: 2024-12-04
```
Comment (aisgbnok) [inline manifests/g/Genymobile/scrcpy/3.0.1/Genymobile.scrcpy.installer.yaml:24]:
```suggestion
```

File: manifests/g/Genymobile/scrcpy/3.0.1/Genymobile.scrcpy.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Genymobile.scrcpy
+PackageVersion: 3.0.1
+InstallerType: zip
+NestedInstallerType: portable
+ArchiveBinariesDependOnPath: true
```
Comment (aisgbnok) [inline manifests/g/Genymobile/scrcpy/3.0.1/Genymobile.scrcpy.installer.yaml:8]: Just a stupid nitpick with the ordering of `ReleaseDate` that makes no real functional difference.

```suggestion
NestedInstallerType: portable
ReleaseDate: 2024-12-04
ArchiveBinariesDependOnPath: true
```

**Review by aisgbnok — APPROVED** [review]
image

--- 

Aside from a stupid ordering nitpick which is just my ocd (no need to approve/commit), this matches the other versions. Most importantly it installs and works as it should.

**Review by aisgbnok — APPROVED** [review]
<details>
  <summary>Works with manual validation.</summary>

image

</details>

**Review by stephengillie — APPROVED** [review]

**Pull request merged**